devicon.json

```diff
@@ -7509,6 +7509,29 @@
         "color": "#2298BD",
         "aliases": []
     },
+    {
+        "name": "tauri",
+        "altnames": [],
+        "tags": [
+            "rust",
+            "electron",
```

Comment:
I don't think electron should be a tag here, since it's a competitor, and not a category of tauri so to say. Like I wouldn't put C# as a tag under Java, even though they are very similar and fierce competitors
```suggestion
```